web/modules/custom/dpl_webform/dpl_webform.install

```diff
@@ -107,3 +108,89 @@ function dpl_webform_update_10004(): string {
 
   return implode("\n", $feedback);
 }
+
+/**
+ * Set default language to english.
+ *
+ * We have experienced issues with the default site language being set to
+ * danish, which causes issues when doing configuration translations. To
+ * prevent this in the future, we now set the default language to english.
+ */
+function dpl_webform_update_10005(): string {
+  $feedback = [];
+
+  \Drupal::configFactory()->getEditable('system.site')->set('langcode', 'en')->save();
+  \Drupal::configFactory()->getEditable('system.site')->set('default_langcode', 'en')->save();
+
+  return implode("\n", $feedback);
+}
+
+/**
+ * Remove unwanted webform settings from config ignore.
+ *
+ * Some libraries has been able to add unwanted webform settings to the
+ * config ignore list. This is preventing us from fixing some faulty
+ * webform translation configurations. This update hook will remove all
+ * webform.webform_options.* and webform.settings from the ignore list.
+ */
+function dpl_webform_update_10006(): string {
+  $config_factory = \Drupal::configFactory();
+  $config_ignore_auto_settings = $config_factory->getEditable('config_ignore_auto.settings');
+  $ignored_configs = $config_ignore_auto_settings->get('ignored_config_entities');
+
+  // Remove webform.settings from config ignore list.
+  if (($key = array_search('webform.settings', $ignored_configs)) !== FALSE) {
+    unset($ignored_configs[$key]);
```

Comment:
if ($key = array_search('webform.settings', $ignored_configs)) {
    unset($ignored_configs[$key]);

Reply:
Removed redundant !== FALSE check

Reply:
I decided to re-add the !==FALSE statement after testing the code when removing it. It did not work as intended as array_search does not return TRUE if found, but instead either the string itself. Therefore I have to explicit check that the return is not FALSE.